lib/rules/no-extra-parens.js

```diff
@@ -73,6 +75,32 @@ module.exports = {
          * @private
          */
         function ruleApplies(node) {
+            if (node.type === "JSXElement") {
+                const isSingleLine = node.loc.start.line === node.loc.end.line;
+
+                switch (IGNORE_JSX) {
+
+                    // Exclude this JSX element from linting
+                    case "all":
+                        return false;
+
+                    // Exclude this JSX element if it is multi-line element
+                    case "multi-line": {
```

Comment:
Oops, forgot to remove it after a bit of refactoring. I had a `const` inside before.